### ISSUE TITLE
[MdCheckbox] Checkbox boolean model with default value of null

### ISSUE THREAD

**Issue by sambhav-gore** [vuematerial/vue-material#1701]
I want the ability to specify the true-value and false-value as boolean when the initial (default) value for the model for checkbox is null

### Steps to reproduce

```html
<md-checkbox v-model="field">Approved?</md-checkbox>
{{field}}
```

### Which browser?

Chrome

### What is expected?

When the default value is 'null' after checking the checkbox it becomes 'on' 
I want the ability to specify that when the checkbox is checked, the value in model should be 'true' and when further unchecked it should be false.
In other words I want the `v-bind:true-value` and `v-bind:false-value` to work correctly with md-checkbox

### What is actually happening?

md-checkbox is dynamically setting the value in model by checking the type of binding.

**Comment by VdustR** [comment]
I didn't get it. I don't understand why won't you just use `Boolean` for **boolean** model?

**Comment by sambhav-gore** [comment]
@VdustR Correct, but as of now our API responds with value of null for boolean fields. but when sending it back we need to have a boolean. I know it is not ideal as of now.

**Comment by sambhav-gore** [comment]
@VdustR I created this codesandbox with default input to showcase what I am trying to achieve. https://codesandbox.io/s/mm2x5k7jv9 

with md-checkbox I am unable to bind the `true-value` and `false-value` directives

**Comment by VdustR** [comment]
I think [computed](https://vuejs.org/v2/guide/computed.html) is what you want.

**Comment by VdustR** [comment]
Do you mean if the checkbox is checked post a value `'value'` and if it's unchecked post `'false'`?

**Comment by VdustR** [comment]
I got it now. Thanks.

**Comment by sambhav-gore** [comment]
no. If the checkbox is checked then model value is 'true' when unchecked the value is 'false'. However, initially the model value is null. Once again, this is exactly what I want - https://codesandbox.io/s/mm2x5k7jv9

**Comment by Samuell1** [comment]
Use it as 
`<md-checkbox v-model="toggle" value="true">Accept terms? </md-checkbox>`
and toogle default data value make as `false`

**Comment by VdustR** [comment]
@Samuell1 
https://vuejs.org/v2/guide/forms.html#Checkbox-1

We didn't support `true-value` and `false-value` yet. That's what he ask.

@sambhav-gore am I right?

**Comment by sambhav-gore** [comment]
@VdustR That is exactly right. Need support for `true-value` and `false-value` .

**Comment by sergiopani** [comment]
The default value of a checkbox is undefined, when you click it it puts on True and when you ckick again it puts on False.
